src/app/me/sources.ts

```diff
@@ -0,0 +1,16 @@
+import { DataSourceResponse } from '@/app/application/services/data-source/DataSourcePool'
+
+type Closeable = { close: () => void }
+
+export type MeSource = DataSourceResponse<{
```

Comment:
**Question**: Why the name? What does it mean?

Reply:
Good/commonly used name for profile/user setting type things. Pretty much guaranteed not to clash with anything else that may be added to to kuma in the future (such as `config`, `settings`). I think most dev folks would think "`/me` oh, this is probably about user profile/settings"?

Reply:
I’d prefer something like `UserConfigSource` to make this unambiguous.

Reply:
What if its not configurable, say we want the browser the user is using? `UserConfigSource` breaks down. What if we end up with a concept of multiple users? `UserConfigSource` breaks down.

`/me` is generic enough to cover anything we need this for without naming issues, plus I'd say its pretty recognisable to most folks at a glance.